### PR TITLE
Package fix 1

### PR DESCRIPTION
**Pull request by zacsimile**
Addresses issue # package installs.

**Is this a bugfix or an enhancement?**
bugfix

**Proposed changes:**
- Update instructions based on use of latest `constructor` and `conda-build` packages. Remove use of local packages since we pull them from the `david_baddeley` channel anyway.
- Shift spec determination onto `pyme-depends`
- Disable `keep_pkgs` (no need)
- Install dependencies in order to remove errors thrown during install
- Convert `http` to `https` (for peace of mind)

**Checklist:**

- [x] Tested with numpy=1.14
- [ ] Tested on python 2.7 and 3.6
- [ ] Tested with wx=3.x and wx=4.x [if UI code]
- [ ] Does the PR avoid variable renaming in existing code, whitespace changes, and other forms of tidying? [There is a place for code tidying, but it makes reviewing 
much simpler if this is kept separate from functional changes]

If an enhancement (or non-trivial bugfix):

- [ ] Has this been discussed in advance (feature request, PR proposal, email, or direct conversation)?
- [ ] Does this change how users interact with the software? How will these changes be communicated?
- [ ] Does this maintain backwards compatibility with old data?
- [ ] Does this change the required dependencies?
- [ ] Are there any other side effects of the change?

### CODE REVIEW

**Comment by zacsimile** [comment]
Next steps are to accept this, python-microscopy/pyme-conda-recipes#7 and python-microscopy/pyme-conda-recipes#8, `conda-build` the packages with the latest `conda-build`, and push the new packages to conda/david_baddely. At this point, the new install instructions with `constructor>=3.0.0` installed from the `conda` channels in a fresh environment should produce a perfectly-working installer on Windows, and likely will on OSX (although I'm still not 100% sure the framework error will be fixed, though it should be).

Getting these installers signed is a separate issue. We can sign them ourselves and attach the signature on OSX with `signing_identity_name` in `constructor.yaml`. Since this would not use a recognized signing authority, users would have to accept our signature the first time they install PYME. As long as we use the same signature file, all future PYME should be automatically accepted. I could not find clear instructions on how to sign our Windows installer. 

We should consider an internal upgrade call in the `Help` menu. Can just call `conda update` in the environment.

File: docs/dev/executable_installers.md [hunk 1]
```diff
@@ -3,22 +3,14 @@
...
-4) Download the latest `conda.exe` for the platform you are building for from 
-[https://repo.anaconda.com/pkgs/misc/conda-execs/](https://repo.anaconda.com/pkgs/misc/conda-execs/) see also 
-[https://github.com/conda/constructor/issues/257](https://github.com/conda/constructor/issues/257)
-5) Change to the `python-microscopy` directory
-6) Edit `construct.yaml` to set the version to the latest conda package version
-6) Run `constructor --conda-exe PATH_TO_CONDA.EXE .`
+Instructions work for `miniconda3`.
+
+1) Create a clean conda environment (e.g. `conda create -n pyme_build python=2.7`)
+2) Activate the new environment and run `conda install instructor`. This should install `constructor>=3.0.0`.
```
Comment (David-Baddeley) [inline docs/dev/executable_installers.md]: instructor???

File: construct.yaml [hunk 2]
```diff
@@ -35,9 +36,9 @@ exclude:
 packages:
 #  - python-2.7.9-0.tar.bz2
 
-#install_in_dependency_order: True
+install_in_dependency_order: True
 
-keep_pkgs: True
+# keep_pkgs: True
```
Comment (David-Baddeley) [inline construct.yaml:40]: why?

File: construct.yaml [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
+  - https://repo.anaconda.com/pkgs/free/
   - https://conda.anaconda.org/david_baddeley
-  - http://repo.anaconda.com/pkgs/msys2
+  - https://repo.anaconda.com/pkgs/msys2
 
 # specifications
 specs:
-  - python
+  # - python
+  - pyme-depends
```
Comment (David-Baddeley) [inline construct.yaml]: is a dep of python-microscopy - should not be needed.

File: docs/dev/executable_installers.md [hunk 4]
```diff
@@ -3,22 +3,14 @@
...
-3) `conda install constructor` **NOTE:** At one point it was neccesary to git clone the constructor source, checkout the "v3" branch
-   and do a `python setup.py install` instead of the above. I think that the v3 branch has now been merged into main, but 
-   it might be worth trying the source install if you run into issues.
-4) Download the latest `conda.exe` for the platform you are building for from 
-[https://repo.anaconda.com/pkgs/misc/conda-execs/](https://repo.anaconda.com/pkgs/misc/conda-execs/) see also 
-[https://github.com/conda/constructor/issues/257](https://github.com/conda/constructor/issues/257)
-5) Change to the `python-microscopy` directory
-6) Edit `construct.yaml` to set the version to the latest conda package version
-6) Run `constructor --conda-exe PATH_TO_CONDA.EXE .`
+Instructions work for `miniconda3`.
```
Comment (David-Baddeley) [inline docs/dev/executable_installers.md]: which version??? - my stuff above was also with miniconda3

Comment (zacsimile) [reply docs/dev/executable_installers.md]: conda 4.8.3

File: construct.yaml (hunk 3 above)
Comment (zacsimile) [reply construct.yaml]: Fair enough. We'll delete that, too.

File: construct.yaml (hunk 2 above)
Comment (zacsimile) [reply construct.yaml:40]: It takes up extra space on the user's computer, and the installer works without this flag.

File: docs/dev/executable_installers.md (hunk 1 above)
Comment (zacsimile) [reply docs/dev/executable_installers.md]: Typo, `constructor`.

File: docs/dev/executable_installers.md (hunk 4 above)
Comment (zacsimile) [reply docs/dev/executable_installers.md]: The key is `conda install constructor` now pulls version `3.0.0`, which it should do on older versions of `conda` now as well.

**Comment by zacsimile** [comment]
I think everything here is deprecated at this point.

**Pull request closed without merging**